client/components/checkout/InStock.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const InStock = ({ availableQuantity, city }) => (
+  <div>
+    <div className="col-left">
+      <div className="h-text-bold h-text-green">Pick up today</div>
+      <div className="h-text-bold">{`at ${city}`}</div>
+      <div>Edit store</div>
+      <div className="h-text-orange">{`only ${availableQuantity} left`}</div>
```

Comment:
Suggestion: for class names in each of these files, you may want to consider naming them more specifically around their functions (e.g. h-city-text-bold vs h-text-bold, or something that incorporates the intended role of that class aside from styling).

Up to you! I thought it might make it easier when looking at your tests later, when you're doing a wrapper.find for a specific class.